### PR TITLE
Fix :last-child selector in Web Components and LIT

### PR DESCRIPTION
**Pull request by lpardosixtosMs**
Currently the web components CSS uses  the `.todo-item:last-child` selector, which matches all the `.todo-item` elements since each is the only child of its host. Change by `host(:last-child) > .todo-item`. Similarly, replace `.editing:last-child` with `host(:last-child) .editing` in the LIT todoMVC CSS.

### CODE REVIEW

**Comment by lpardosixtosMs** [comment]
@flashdesignory Can you take a look? Thanks!

**Review by flashdesignory — APPROVED** [review]
approved, assuming it was also tested in the standalone versions.

**Comment by lpardosixtosMs** [comment]
@flashdesignory would you mind merging it? I don't have write access.

> approved, assuming it was also tested in the standalone versions.

Yes, this is a fix mainly for standalone versions:

Before:

image

After:

image

**Pull request merged**
